src/task2/controller/userController.js

```diff
@@ -0,0 +1,76 @@
+const sharedConstants = require('./../util/sharedConstants');
+const userValidator = require('./../validator/userValidator');
+const { createUser, getUser, updateUser, deleteUser, getAutoSuggestUsers } = require('../service/userService');
+const CREATE_OPERATION = 'creating';
+const UPDATE_OPERATION = 'updating';
+const DELETE_OPERATION = 'deleting';
+const GET_OPERATION = 'retrieving';
+
+async function addUser(req, res) {
+    const requestBody = req.body;
+    const validationResult = userValidator.validateCreateRequest(requestBody);
```

Comment:
validations ideally should be done in routes domain

e.g.
```
router.route('/add')
  .post(validator.body(validation.connectUsersToGroup.body), userGroupController.addUsersToGroup);
```

please look at https://www.npmjs.com/package/express-joi-validation